src/wp-admin/edit.php

```diff
@@ -376,7 +376,7 @@
 	/* translators: %s: Number of pages. */
 	'deleted'   => _n( '%s page permanently deleted.', '%s pages permanently deleted.', $bulk_counts['deleted'] ),
 	/* translators: %s: Number of pages. */
-	'trashed'   => _n( '%s page moved to the Trash.', '%s pages moved to the Trash.', $bulk_counts['trashed'] ),
+	'trashed'   => _n( '%s page moved to the Trash.', '%s pages moved to the Trash.', $bulk_counts['trashed'] ) . sprintf( __( 'It will be deleted from trash after %s days.' ), EMPTY_TRASH_DAYS ),
```

Comment:
```suggestion
'trashed'   => _n( '%s page moved to the Trash.', '%s pages moved to the Trash.', $bulk_counts['trashed'] ) . _n( 'It will be deleted from the trash after %s day.', 'It will be deleted from the trash after %s days.', EMPTY_TRASH_DAYS ),
```